source/docs/10-analysisharm.xml

```diff
@@ -297,7 +297,7 @@
                         <specDesc key="att.meterConformance.bar" atts="metcon"/>
                      </specList>
                   </p>
-                  <p>When used on <gi scheme="MEI">staff</gi> or <gi scheme="MEI">layer</gi> elements, the <att>metcon</att> attribute can be used to indicate if the duration of the contained events is equal to ("c"), less than ("i"), or greater than ("o") that predicted by the time signature. When used on the <gi scheme="MEI">measure</gi> element, <att>metcon</att> takes a boolean value, where "true" indicates conformance by all staff and layer descendants of the measure.</p>
+                  <p>When used on <gi scheme="MEI">staff</gi> or <gi scheme="MEI">layer</gi> elements, the <att>metcon</att> attribute can be used to indicate if the duration of the contained events is equal to ("c"), less than ("i"), or greater than ("o") that predicted by the time signature. When used on the <gi scheme="MEI">measure</gi> element, <att>metcon</att> takes a boolean value, where <val>true</val> indicates conformance by all staff and layer descendants of the measure.</p>
```

Comment:
"c", "i", and "o" with val tags? Like "true"

Reply:
Indeed. Thanks for spotting this one.